RadixWallet/Clients/SubmitTransactionClient/SubmitTransactionClient+Live.swift

```diff
@@ -52,38 +52,9 @@ extension SubmitTransactionClient: DependencyKey {
 		let submitTransaction: SubmitTransaction = { request in
 			let txID = request.txID
 
-			func debugPrintTX(_ decompiledNotarized: NotarizedTransaction) {
```

Comment:
quite a few methods can be dropped if we implement this debug in Sargon, which is nice.